content/v2.0/influxdb-templates/create.md

```diff
@@ -79,6 +79,34 @@ influx pkg export all \
   -t $INFLUX_TOKEN
 ```
 
+#### Export resources filtered by labelName or resourceKind
+The `influx pkg export all` command has an optional `--filter` flag that filters
+only exports resources that match a specified `labelName` or `resourceKind`.
+Provide multiple filters for both `labelName` and `resourceKind`
+
+###### Export only dashboards and buckets with specific labels
+The following example exports resources that match this predicate logic:
+
+```js
+(resourceKind == "Bucket" or resourceKind == "Dashboard")
```

Comment:
I'm not sure what your question is.

Reply:
I thought `resourceKind` may be a typo....thought I saw `resourcesTypes` somewhere? Curious why we'd use that term (for example, visualization kinds instead of visualization types).

Reply:
@kelseiv the term `kind` is borrowed from the kubernetes community. Our templates follow a standard setup that the kubernetes community follows.